lib/tests/streamlit/util_test.py

```diff
@@ -72,3 +72,20 @@ def f():
             return True
 
         self.assertEqual(True, hasattr(f, "__wrapped__"))
+
+    @parameterized.expand(
+        [
+            ({}, {}),
+            (
+                {
+                    "HELLO": 4,
+                    "Hello": "world",
+                    "hello": 5.5,
+                },
+                {"hello": 4, "hello": "world", "hello": 5.5},
+            ),
```

Comment:
I think you'll want to use keys that end up being different in the expected dict since in this particular test
`{"hello": 4, "hello": "world", "hello": 5.5} == {"hello": 5.5}`, so the test still passes but probably doesn't quite work as intended